superset/db_engine_specs/hive.py

```diff
@@ -200,7 +200,7 @@ def extract_error_message(cls, e):
         match = re.search(r'errorMessage="(.*?)(?<!\\)"', msg)
         if match:
             msg = match.group(1)
-        return msg
+        return f"Hive error: {msg}"
```

Comment:
I would much prefer to see this in `base.py` along the lines of `f"{self.engine} error: {msg}"` for uniformity and only the engine-specific message extraction logic where necessary.

Reply:
I was thinking a similar thing, having each place that returns an error go through an error formatting function that prepends the engine